backend/src/main/java/com/redhat/cloud/notifications/routers/EndpointReadyChecker.java

```diff
@@ -79,6 +84,13 @@ public void execute() {
             try {
                 CamelProperties cp = em.find(CamelProperties.class, ep.getId()); // TODO Fetch in one go
                 processorId = cp.getExtras().get("processorId");
+
+                // TODO Temporary code meant to clean the DB from stage only. Remove it as soon as it's been executed once.
+                if ("stage".equals(environment) && processorId == null) {
+                    em.remove(ep);
+                    continue;
+                }
```

Comment:
Because of my tests on stage, a few Slack endpoints lost their `processorid` field in the DB. They can't be updated anymore and are stuck forever in this job. This will remove them from the DB.